docs/tutorials/tut-design.md

```diff
@@ -159,7 +159,7 @@ Even if XSOAR doesn't make any distinction, in this tutorial we group parameters
 These parameters determine how Cortex XSOAR connects to the third party API. 
 
 Usually there are at least two parameters:
- - **url**: the URL Cortex XSOAR should connect to. In case the API is SaaS only and the URL is the same for all customers and is never going to change over time, you can omit this parameter and store the URL directly in the code. Some SaaS products have different urls based on the tenant name, for example `customer1.my-test-saas-service.info` and `customer2.my-test-saas-service.info`: in this case we still recommend to use the full url here and not just the tenant name (i.e. have the clients input `customer1.my-test-saas-service.info` and not just `customer1`).
+ - **url**: the URL Cortex XSOAR should connect to. Always expose this parameter even in SaaS applications since SaaS applications can have variations as well (tenant name or federal cloud)

```

Comment:
```suggestion
 - **url**: the URL Cortex XSOAR should connect to. Always expose this parameter even in SaaS applications since SaaS applications can have variations as well (tenant name or federal cloud).

```